Examples/BinaryBH/BinaryBHLevel.cpp

```diff
@@ -102,7 +102,8 @@ void BinaryBHLevel::specificPostTimeStep()
 {
     CH_TIME("BinaryBHLevel::specificPostTimeStep");
 
-    bool first_step = (m_time == m_dt);
+    bool first_step = (m_time == 0.); // called at t=0 from Main
```

Comment:
Comment here could be more informative - e.g. "This form is used when the function is called during setup as well as evolution" etc.